algorithms/ar-bsrh/javascript/bst.js

```diff
@@ -0,0 +1,152 @@
+class Node {
+    constructor(data) {
+        this.right = null;
+        this.left = null;
+        this.data = data;
+    }
+}
+
+class Bst {
+    constructor() {
+        this.root = null;
+    }
+
+    insert(data) {
+        var node = new Node(data);
+        if (!this.root) {
+            this.root = node;
+            return this;
+        }
+        let current = this.root;
+        while (current) {
+            if (data === current.data) {
+                return;
+            }
+            if (data < current.data) {
+                if (!current.left) {
+                    current.left = node;
+                    break;
+                }
+                current = current.left;
+            }
+            if (data > current.data) {
+                if (!current.right) {
+                    current.right = node;
+                    break;
+                }
+                current = current.right;
+            }
+        }
+        return this;
+    }
+    
+    find(data) {
+        if (!this.root) {
+            return null;
+        }
+        let current = this.root;
+        while (current) {
+            if (data == current.data) {
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6264809)